service/deployer/eventer/github/client.go

```diff
@@ -0,0 +1,163 @@
+package github
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+const (
+	// deploymentUrlFormat is the string format for the GitHub
+	// API call to fetch Deployments.
+	deploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments"
+
+	// deploymentStatusUrlFormat is the string format for the
+	// GitHub API call to post Deployment Statuses.
+	deploymentStatusUrlFormat = "https://api.github.com/repos/%s/%s/deployments/%v/statuses"
```

Comment:
Not in the URL, strangely. They seem to use headers to switch on new features as they bring them in, and deprecate loudly. This is the way.